.circleci/config.yml

```diff
@@ -90,3 +117,44 @@ jobs:
             mkdir -p $HOME/testground/plans
             ln -s $HOME/project/plans/placebo $HOME/testground/plans/placebo
             go test -v ./...
+
+  # Tests the k8s runner can successfully submit jobs.
+  kind-test:
+    executor: linux
+    steps:
+      - setup-install-bins
+      - run:
+          name: "build testground"
+          command: make install
+      - run:
+          name: "configure cluster"
+          command: |
+            kind create cluster --wait 90s
+            kubectl apply -f .circleci/pv.yaml
```

Comment:
Do we want these somewhere else?
These are used to create the fake EFS volume in our fake kubernetes cluster.

Reply:
For now they are fine here I think. Ideally we want the test setup to be at least surprising as possible with very little indirection.